iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/ArmTree.java

```diff
@@ -58,6 +58,8 @@
 import org.sonar.iac.arm.tree.api.bicep.typed.TypedVariableBlock;
 import org.sonar.iac.arm.tree.api.bicep.variable.LocalVariable;
 import org.sonar.iac.arm.tree.api.bicep.variable.VariableBlock;
+import org.sonar.iac.arm.tree.impl.bicep.importdecl.ImportAsClause;
+import org.sonar.iac.arm.tree.impl.bicep.importdecl.ImportWithClause;
```

Comment:
Why do we need to import `tree.impl` here compared to the `tree.api` for the rest?
I see that these tree classes are not implemented in this sprint, but do you know, why there are no interfaces for them?

Reply:
I don't know why interfaces are missing, the PR: https://github.com/SonarSource/sonar-iac/pull/799
I will add interfaces.

Reply:
I think it was my implementation; these elements seemed like only intermediate and not as important as the entire `import` statement. I believe I didn't foresee that they will also be used in the future.

Reply:
It is fixed.